components/tooltip/README.md

```diff
@@ -112,24 +112,32 @@ you wish to change and include them after the default styles.
 
 **You can put any custom styles inline or use class names and add an external file.**
 
-
 ## How to use
 
-Usually a tooltip is displayed as a result of an action performed on another element. For example you need to show a tooltip with more data when you hover on an item in the inventory. To specify the trigger element use the `target` attribute. Its value is a query string used to get the element. Refer to the table below for the full list of attributes.
+Usually a tooltip is displayed as a result of an action performed on another element. For example, you need to show a tooltip with more data when you hover on an item in the inventory. To specify the trigger element use the `target` attribute. Its value is a query string used to get the element. Refer to the table below for the full list of attributes.
+
+Async mode use case is shown in the demo page.

```

Comment:
Can we describe somewhere in the documentation that it is required for the `setMessage` to receive a callback that returns a promise when the tooltip has an async attribute?

Reply:
Done.